File: 2298-count-integers-with-even-digit-sum/2298-count-integers-with-even-digit-sum.py
class Solution:
    def countEven(self, num: int) -> int:
        c = 0
        for i in range(1,num+1):
            s = 0
            for k in str(i):
                s = s + int(k)
            if s % 2 == 0 :
                c += 1
        return c

        

File: 2298-count-integers-with-even-digit-sum/2298-count-integers-with-even-digit-sum.c
int countEven(int num)
 {
   int c,m,s,i;

   c=0;
   for(i=1;i<=num;i++)
   {
    m=i;
    s=0;
     while(m!=0)
    {
     s=s+(m%10);
     m=m/10;
    }

  if(s%2==0)
   c++;

   }

  return c;     
}